featurebyte/service/feature_preview.py

```diff
@@ -0,0 +1,335 @@
+"""
```

Comment:
Split this from preview.py to handle catalog-specific preview (feature, featurelist)